IntercomDotnet/Client.cs

```diff
@@ -15,9 +15,9 @@ public class Client
         public string UserName { get; set; }
         public string Password { get; set; }
 
-        public double GetUnixTimestamp(DateTime inputTime)
+        public long GetUnixTimestamp(DateTime inputTime)
         {
-            return (inputTime - new DateTime(1970, 1, 1).ToUniversalTime()).TotalSeconds;
+            return (long)inputTime.ToUniversalTime().Subtract(new DateTime(1970, 1, 1, 0, 0, 0)).TotalSeconds;
```

Comment:
Yeah time zones are messy :)

I'm currently in a non-UTC time zone (UTC+2) and I have tried the test program with the following cases:

```client.Events.Post("test_event", DateTime.Now, "userid", new { });```
```client.Events.Post("test_event", DateTime.UtcNow, "userid", new { });```

Both show up with the (same) correct time in Intercom's UI (localized to my time zone). 

So for example if ```DateTime.UtcNow``` is ```2015-07-29 09:04:40```. Then the time showing in Intercom is ```2015-07-29 11:04:40```, which is my local time. The unix time sent to Intercom is 1438160680 (which corresponds to the UTC time ```2015-07-29 09:04:40```)

Unix time is defined as the number of seconds since 1970. So I think that to get the UTC time ```2015-07-29 09:04:40``` as unix time, you need to calculate the seconds between ```1970-01-01 00:00:00``` and ```2015-07-29 09:04:40```.

```new DateTime(1970, 1, 1).ToUniversalTime()``` would give you ```1969-12-31 22:00:00``` (depending on your time zone) and would be incorrect to use as a base when calculating the number of seconds since 1970.

The advantage with doing ```ToUniversalTime()``` on ```inputTime``` is that the user of the library can use either ```DateTime.Now``` or ```DateTime.UtcNow```. The unix timestamp sent to Intercom would always be corrert UTC time, since ```ToUniversalTime()``` on an UTC ```DateTime``` does nothing.

Hm. Long explanation of how I'm thinking. Not sure I've made it very clear... ;) I hope you understand!